sync/class.jetpack-sync-defaults.php

```diff
@@ -322,6 +322,25 @@ public static function get_post_meta_whitelist() {
 		return apply_filters( 'jetpack_sync_post_meta_whitelist', self::$post_meta_whitelist );
 	}
 
+	static $user_meta_whitelist = array(
+		'locale',
+		'*_user_level',
```

Comment:
Maybe we could whitelist `$wpdb->get_blog_prefix() . 'user_level'` and $wpdb->get_blog_prefix() . 'capabilities' ?

Reply:
I updated it so that when every-time we get the whitelist we return the values with the prefixed defined.